nilearn/mass_univariate/_utils.py

```diff
@@ -0,0 +1,81 @@
+"""Utility functions for the permuted least squares method."""
+import nibabel as nib
+import numpy as np
+from scipy import ndimage
+from nilearn.masking import apply_mask, unmask
+
+
+def _calculate_tfce(scores_array, masker, E=0.5, H=2, dh=0.1):
+    """Calculate threshold-free cluster enhancement values for scores maps.
+
+    The TFCE calculation is implemented as described in [1]_.
```

Comment:
I feel like it could be worth adding `TFCE` to the glossary. I personally had no idea what it meant before reading the docstring of this private function. WDYT?

Reply:
I've added an entry. Please let me know if you think it needs more info.

Reply:
Thanks for adding it! I think you could even add [your link](https://benediktehinger.de/blog/science/threshold-free-cluster-enhancement-explained/) in the glossary entry.

Reply:
Done!